src/pages/Pick/Pick.js

```diff
@@ -138,43 +146,78 @@ export class Pick extends Component {
 
   handlePage = (targetPage) => {
     let newPageArr = ["<<", "<", 1, 2, 3, 4, 5, 6, 7, ">", ">>"];
-    const resetColor= ["PageButton","PageButton", "PageButton", "PageButton", "PageButton", 
-    "PageButton", "PageButton", "PageButton", "PageButton","PageButton", "PageButton"];
-    let pageIdxBtnColor = ["PageButton","PageButton", "PageButton", "PageButton", "PageButton", 
-    "PageButton", "PageButton", "PageButton", "PageButton","PageButton", "PageButton"];
+    const resetColor = [
+      "PageButton",
+      "PageButton",
+      "PageButton",
+      "PageButton",
+      "PageButton",
+      "PageButton",
+      "PageButton",
+      "PageButton",
+      "PageButton",
+      "PageButton",
+      "PageButton",
+    ];
+    let pageIdxBtnColor = [
+      "PageButton",
+      "PageButton",
+      "PageButton",
+      "PageButton",
+      "PageButton",
+      "PageButton",
+      "PageButton",
+      "PageButton",
+      "PageButton",
+      "PageButton",
+      "PageButton",
+    ];
 
-    if (targetPage >= 4){
+    if (targetPage >= 4) {
       newPageArr = [
-        "<<", "<", targetPage-3, targetPage-2, targetPage-1,
-        +targetPage, +targetPage+1, +targetPage+2, +targetPage+3, ">", ">>"
+        "<<",
+        "<",
+        targetPage - 3,
+        targetPage - 2,
+        targetPage - 1,
+        +targetPage,
+        +targetPage + 1,
+        +targetPage + 2,
+        +targetPage + 3,
+        ">",
+        ">>",
       ];
       pageIdxBtnColor.splice(5, 1, "PageButton clicked");
-    } else if(Number(targetPage) === 3){
-      pageIdxBtnColor.splice(4, 1, "PageButton clicked")
-    } else if(Number(targetPage) === 2){
+    } else if (Number(targetPage) === 3) {
+      pageIdxBtnColor.splice(4, 1, "PageButton clicked");
+    } else if (Number(targetPage) === 2) {
       pageIdxBtnColor.splice(3, 1, "PageButton clicked");
-    } else if (targetPage === 1 || "<<"){
+    } else if (targetPage === 1 || "<<") {
       pageIdxBtnColor.splice(2, 1, "PageButton clicked");
-    }  
-    
-    if (targetPage ===  "<"){
-      const beforePageArr = this.state.pageArr.slice(2,9).map((num)=>(Number(num)-1));
-      newPageArr= ["<<", "<", ...beforePageArr, ">", ">>"];
+    }
+
+    if (targetPage === "<") {
+      const beforePageArr = this.state.pageArr
+        .slice(2, 9)
+        .map((num) => Number(num) - 1);
```

Comment:
```suggestion
        .map((num) => +num - 1);
```
string을 number로 변경해줄 시 이런식의 변경도 가능하겠네요!

Reply:
팀원분께 전달하겠습니다...!!